platform-modules/batch-models/src/main/scala/org/ekstep/analytics/job/report/StateAdminReportHelper.scala

```diff
@@ -50,6 +53,7 @@ trait StateAdminReportHelper extends  BaseReportsJob {
         col("blockname").as("Block name"),
         col("slug").as("slug"),
         col("externalid"))
+    blockData.show(10,false)
```

Comment:
Remove this print statement

Reply:
print line is removed. Added the column check for index also.